@kiva/kv-components/vue/KvPagination.vue

```diff
@@ -109,7 +109,7 @@ export default {
 	},
 	computed: {
 		current() {
-			const page = this.offset / this.limit;
+			const page = Math.floor(this.offset / this.limit) + 1;
```

Comment:
Would this be the same as Math.ceil?

Reply:
yeah, and the best to pass the test, ty!